n3fit/src/n3fit/tests/test_msr.py

```diff
@@ -6,9 +6,11 @@
 
 def apply_layer_to_fixed_input(layer):
     np.random.seed(422)
-    pdf_integrated = op.numpy_to_tensor(np.random.normal(size=(1, 14)))
+    pdf_integrated = np.expand_dims(np.random.normal(size=(1, 14)), -1)
+    pdf_integrated = op.numpy_to_tensor(pdf_integrated)
```

Comment:
I agree, I just did it like that because I thought your way would change the numerics and I'd have to update them, something I've had issues with before due to different numpy versions I think. Not sure if they will change, so I'll change it and we'll see.

Reply:
Seems to work! :)